contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -311,14 +316,16 @@ contract ArbitrumValidator is TypeAndVersionInterface, AggregatorValidatorInterf
   }
 
   /**
-   * @notice Internal method that approximates the `maxSubmissionCost` (using the L1 gas price feed)
+   * @notice Internal method that approximates the `maxSubmissionCost`
```

Comment:
Please don't remove docs, but replace them with new docs. What is the new method we are using to approximate the `maxSubmissionCost`?

Reply:
Do we need to add buffer (was 200% on their formula before), why not? 
Please document those decisions.

Reply:
this was removed as we are no longer using the L1 gas price feed.  We are instead calling a function exposed by the Arbitrum team to calculate `maxSubmissionCost`.

Reply:
added more comments to explain this.